charts/vh-booking-queue-subscriber/values.dev.template.yaml

```diff
@@ -3,13 +3,18 @@ java:
   image: '${IMAGE_NAME}'
   releaseNameOverride: ${RELEASE_NAME}
   environment:
-    QUEUENAME: booking
+    QUEUENAME: shaed-dev
+    VHSERVICES__BOOKINGSAPIURL: https://vh-bookings-api-pr-938.dev.platform.hmcts.net/
+    VHSERVICES__VIDEOAPIURL: https://vh-video-api-pr-711.dev.platform.hmcts.net/
+    VHSERVICES__USERAPIURL: https://vh-user-api-pr-259.dev.platform.hmcts.net/
+    VHSERVICES__NOTIFICATIONAPIURL: https://vh-notification-api-pr-173.dev.platform.hmcts.net/
+    VHSERVICES__VIDEOWEBURL: https://vh-video-web-pr-2354.dev.platform.hmcts.net/
 function:
   releaseNameOverride: ${RELEASE_NAME}
   triggers:
   - type: azure-servicebus 
     serviceBusName: "vh-infra-core-dev"
-    queueName: booking
+    queueName: shaed-dev
```

Comment:
just a reminder to revert this and update packages to latest when you're finished